react/src/components/LiveTable.test.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import LiveTable from './LiveTable';
+
+it('renders without crashing', () => {
```

Comment:
What sort of things can we do with tests? Can we test that a certain number of elements are in the table? Compare certain rows to an expectation? I can think of a number of useful things we could do especially on the Leaderboard pages (as it's by far the most complicated).

Reply:
Also what's the difference between a .test.js component and things that you'd put in the tests directory you made?

Reply:
For part one - the answer is yes to basically all of them. It might be worth testing the API call to make sure we are indeed getting 224 rows back, or a multiple of 14 back (if we're doing old years too). But the leaderboard is a prime area for testing, I was thinking the same thing.

As for the .test.js component vs the tests directory, it's very much a style decision for me, but I usually put end-to-end tests and integration tests in the tests folder, and then unit/component-specific tests right along with the component. I'm 100% fine with putting all component tests in the tests folder too.